e2e/react-core/src/react-module-federation.test.ts

```diff
@@ -141,6 +141,17 @@ describe('React Module Federation', () => {
     // check default generated host is built successfully
     const buildOutput = runCLI(`run ${shell}:build:development`);
     expect(buildOutput).toContain('Successfully ran target build');
+
+    // check serves devRemotes ok
+    const shellProcess = await runCommandUntil(
+      `serve ${shell} --devRemotes=${remote} --verbose`,
+      (output) => {
+        return output.includes(
+          `All remotes started, server ready at http://localhost:4200`
```

Comment:
Not a blocking request, but can we try to keep the ports interpolated in these tests and have them somewhere explicitly in the const?

This helps with future port overlap issues.

Reply:
The host and remote generators do not have an option for specifying the port, this is a potential enhancement we should add to those generators